packages/web-config-server/src/apiV1/dashboardses.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { RouteHandler } from './RouteHandler';
+import { NoPermissionRequiredChecker } from './permissions';
+
+const sortByDbColumns = (array, columns) => {
+  // Sorts an array of objects by properties in a prioritized order
+  let sortedOutput = [];
+  columns.forEach(columnName => {
+    sortedOutput = sortedOutput.concat(array
+      .filter(item => !!item[columnName])
+      .sort((itemA, itemB) => itemA[columnName] > itemB[columnName] ? 1 : -1));
+    array = array.filter(item => !item[columnName]);
+  });
+  return [...sortedOutput, ...array];
+}
+
+export default class extends RouteHandler {
+  static PermissionsChecker = NoPermissionRequiredChecker;
+
+  buildResponse = async () => {
+    const { entity, query } = this;
+    const { code: entityCode } = entity;
+    const userGroups = await this.req.getUserGroups(entityCode);
```

Comment:
```suggestion
    const permissionGroups = await this.req.getUserGroups(entityCode);
```
It might be too hard to make the `this.req.getUserGroups` consistent, but at least we can use `permissionGroups` everywhere new